app/src/androidTest/java/app/wefridge/wefridge/LocationControllerUnitTest.kt

```diff
@@ -0,0 +1,40 @@
+package app.wefridge.wefridge
+
+import android.content.Context
+import app.wefridge.wefridge.model.LocationController
+import com.google.firebase.firestore.GeoPoint
+import org.junit.Test
+import androidx.test.platform.app.InstrumentationRegistry
+import org.junit.Assert.assertEquals
+import org.junit.runner.RunWith
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.Before
+
+
+/*
+* NOTE: Run the following tests with device language set to English.
+* */
+@RunWith(AndroidJUnit4::class)
+class LocationControllerUnitTest {
```

Comment:
Does it make sense for us to test the Android GeoCoder? We didn't make it ourselves, so we have correspondingly little control.

Reply:
That's right. Shall I remove the tests for that? @frcroth what is your opinion on that?

Reply:
You mean the whole file? No I think it can stay

Reply:
Okay, then I will leave the file in the project for now.